RELEASE_NOTES.rst

```diff
@@ -146,6 +165,10 @@ Bug Fixes
 
 Misc/Internal
```

Comment:
For later discussion
I don't think `Misc/Internal` is good title.
If it's Internal then we are marking with `changelog:skip` and it will not appear in the change log.

Reply:
Agree. `Misc` should be for those changes that do not introduce new features or bug fixes - but can have impact on what the user experiences. Good example is bumping version of FAB or Pydantic - both of which are neither features nor bug fixes, but they might cause some issues for the user and the user needs to be able to learn from the release notes that "Something" has changed. Similalry daskexcutor removal or `pyarrow_hotfix`.

So I think the only problem here is the name (which IMHO should be just `Misc`).

Reply:
Let me change it today :)